intranet/static/js/eighth/signup.js

```diff
@@ -63,14 +64,15 @@ $(function() {
             var target = e.target;
             console.log(target);
             var container = $("#roster-section");
-            if ($.trim(container.html())=='') {
+
+            if ($.trim(container.html()) === '') {
                 var spinnerEl = document.getElementById("signup-spinner");
                 var spinner = new Spinner(spinnerOptions).spin(spinnerEl);
                 var schact_id = this.model.attributes.scheduled_activity.id;
                 console.debug("Load roster for scheduled activity", schact_id);
                 var endpoint = $(target).data("endpoint");
-                $.get(endpoint + "/" + schact_id, {}, function(resp) {
-                    container.html(resp);
+
+                container.load(endpoint + "/" + schact_id, {}, function(resp) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'schact_id' is not in camel case.](https://www.codacy.com/app/tjhsst-ion/ion/file/2743150002/issues/source?bid=3212528&fileBranchId=3355681#l75)